org.siril.Siril.json

```diff
@@ -0,0 +1,288 @@
+{
+    "app-id": "org.siril.Siril",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "command": "siril",
+    "rename-icon": "org.siril.Siril",
+    "rename-mime-file": "siril.xml",
+    "rename-mime-icons": [
+        "text-x-seq"
+    ],
+    "provides": {
+        "id": "org.free_astro.siril"
+    },
+    "replaces": {
+        "id": "org.free_astro.siril"
+    },
```

Comment:
This is supposed to be in your metainfo file not here 

https://gitlab.com/free-astro/siril/-/blob/master/platform-specific/linux/org.free_astro.siril.appdata.xml?ref_type=heads

Reply:
Fixed in 57d8a6a